assets/acquia_lift.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file
+ * Contains the definition of the behaviour openAcquiaLiftExperienceBuilder.
+ */
+
+(function ($, Drupal) {
+
+    'use strict';
+
+    /**
+     * Attaches the JS test behavior to to weight div.
+     */
+    Drupal.behaviors.acquiaLiftOpenExperienceBuilder = {
+        attach: function (context, settings) {
+            $('.openLiftLink').click(function() {
```

Comment:
Ugh, this is going to add an event listener for every AJAX callback, because behaviors are attached for every single one of those. So you'll actually want:

```
Drupal.behaviors.acquiaLiftOpenExperienceBuilder = {
  attach: function (context, settings) {
    $(context)
        .find('.toolbar .toolbar-bar .openLiftLink')
        .once('acquia-lift-open-experience-builder')
        .on('click', openExperienceBuilder);
  }
};
```

(This is also why it's great to have the actual logic separate. It doesn't need to change now.)